src/components/organisms/pupil/OakPreviousLessonsHeading/OakPreviousLessonsHeading.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { OakBulletList } from "@/components/molecules";
+import { OakFlex, OakHeading } from "@/components/atoms";
+
+export type OakPreviousLessonsHeadingProps = {
+  numberOfLessons: number;
+  yearDescription: string;
```

Comment:
I wouldn't try to align these with Schema data types. I'd rather just go for year, examboard, subject etc. It's more confusing this way around

Reply:
yep